test/integration/enumerate_databases.test.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../src';
+
+describe('listDatabases', function () {
+  // TODO(NODE3860): Create driver test variants that run with AUTH enabled
+  beforeEach(function () {
+    if (process.env.AUTH !== 'auth') {
+      this.skip();
```

Comment:
Can we make use of the skipReason that Neal implemented in his recent PR to append the reason to the skip?

Reply:
```suggestion
      this.currentTest.skipReason = 'TODO(NODE-3860): requires AUTH enabled';
      this.skip();
```

There's not really docs on this so just fyi this is how you would do it!